lib/testPaymentExpiry.js

```diff
@@ -0,0 +1,191 @@
+'use strict'
+const _ = require('lodash')
+const moment = require('moment')
+const request = require('co-request')
+const UnacceptableExpiryError = require('../errors/unacceptable-expiry-error')
+
+module.exports = function * (config, payment) {
+  let tester = new TransferTester(config, payment)
```

Comment:
`let` should be avoided when `const` suffices as an assurance against mutation. This is the `prefer-const` rule in eslint. I recommend applying our style guide for better static analysis: https://github.com/ripple/javascript-style-guide